packages/jellyfish-transaction-builder/src/txn/txn_builder_account.ts

```diff
@@ -0,0 +1,33 @@
+import { UtxosToAccount } from '@defichain/jellyfish-transaction/dist/script/defi/dftx_account'
+import { OP_CODES, Script, TransactionSegWit } from '@defichain/jellyfish-transaction'
+import { P2WPKHTxnBuilder } from './txn_builder'
+import { TxnBuilderError, TxnBuilderErrorType } from './txn_builder_error'
+
+export class TxnBuilderAccount extends P2WPKHTxnBuilder {
+  /**
+   * Requires at least 0.001 DFI to create transaction, actual fees are much lower.
+   *
+   * @param {UtxosToAccount} utxosToAccount txn to create
+   * @param {Script} changeScript to send unspent to after deducting the (converted + fees)
+   */
+  async utxosToAccount (utxosToAccount: UtxosToAccount, changeScript: Script): Promise<TransactionSegWit> {
+    if (utxosToAccount.to.length !== 1) {
+      throw new TxnBuilderError(TxnBuilderErrorType.INVALID_UTXOS_TO_ACCOUNT_OUTPUT,
+        'Conversion output `utxosToAccount.to` array length must be one'
+      )
+    }
+
```

Comment:
Should also add `balances.token` !== `0x00` since we don't support non `0x00` now.